source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.h

```diff
@@ -11,7 +11,8 @@
 @interface ACOHostConfig:NSObject
 
 @property NSArray<NSString *> *fontFamilyNames;
-
+@property BOOL areAllActionsHaveIcons;
```

Comment:
>areAllActionsHaveIcons [](start = 15, length = 22)

Grammatically, this should be `doAllActionsHaveIcons`, or preferably just `allActionsHaveIcons`.

Reply:
thanks :-)

---
In reply to: [197509368](https://github.com/Microsoft/AdaptiveCards/pull/1641#discussion_r197509368) [](ancestors = 197509368)